src/utils/hooks.js

```diff
@@ -19,3 +24,92 @@ export function useStateWithStorage(storageKey, initialValue) {
 
 	return [value, setValue];
 }
+
+export function useUrgency(items) {
```

Comment:
As we’re introducing this new hook, there’s a great opportunity to enhance our code organization and scalability. I propose that we create a dedicated directory for hooks and another for utility functions. Currently, hooks are placed under the utils directory, but they should be separated into their own directory. Similarly, we should keep utility functions in a distinct utils directory. Let’s ensure each hook is in its own file, while related utility functions are grouped together, maintaining separate files for each hook and utility function where appropriate. This restructuring will improve the maintainability and navigability of our codebase.

Reply:
I moved hooks from hooks.js into relevant files inside hooks folder!